ublox/build.rs

```diff
@@ -1,20 +1,18 @@
 fn main() {
-    #[cfg(all(feature = "ubx_proto23", feature = "ubx_proto27",))]
-    compile_error!(
-        r#"The "ubx_proto23" and "ubx_proto27" features are mutually exclusive and cannot be activated at the same time. Please disable one or the other."#
-    );
-
-    #[cfg(all(feature = "ubx_proto27", feature = "ubx_proto31",))]
-    compile_error!(
-        r#"The "ubx_proto27" and "ubx_proto31" features are mutually exclusive and cannot be activated at the same time. Please disable one or the other."#
-    );
-
-    #[cfg(all(feature = "ubx_proto23", feature = "ubx_proto31",))]
+    #[cfg(any(
+        all(feature = "ubx_proto17", feature = "ubx_proto23"),
+        all(feature = "ubx_proto17", feature = "ubx_proto27"),
+        all(feature = "ubx_proto17", feature = "ubx_proto31"),
+        all(feature = "ubx_proto23", feature = "ubx_proto27"),
+        all(feature = "ubx_proto23", feature = "ubx_proto31"),
+        all(feature = "ubx_proto27", feature = "ubx_proto31")
+    ))]
     compile_error!(
-        r#"The "ubx_proto23" and "ubx_proto31" features are mutually exclusive and cannot be activated at the same time. Please disable one or the other."#
+        r#"The "ubx_protoXX" features ar mutually exclusve and cannot be activated at the same time. Please activate only one at a time."#
```

Comment:
The typo checker workflow should've caught this, why didn't it run?

```suggestion
        r#"The "ubx_protoXX" features are mutually exclusive and cannot be activated at the same time. Please activate only one at a time."#
```

Reply:
Good catch, thanks!

Reply:
yea I think you're right about that